### PR TITLE
maintenance(docs): update go version in readme

### PR DESCRIPTION
**Pull request by noot**
## Changes

- update go version in readme to 1.17

## Tests

```
n/a
```

## Issues

- n/a

## Primary Reviewer

- @qdm12

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ For more information about Gossamer, the Polkadot ecosystem, and how to use Goss
 
 ### Prerequisites
 
-install go version `>=1.16`
+install go version `>=1.17`
```
Comment (qdm12) [inline README.md:46]: I put 1.16 on purpose, I think it should work still.
We use `io` and `embed` from 1.16, however I realised our go.mod got 1.17-updated so maybe it doesn't work with 1.16 :thinking: They sort of broke compatibility with that go.mod change when I think about it...

Comment (noot) [reply README.md:46]: go 1.16 doesn't work anymore as we use `time.UnixMilli` which is only in go 1.17

Comment (qdm12) [reply README.md:46]: Ok so actually, the go.mod still works on 1.16 but we do use `time.UnixMilli` which is only here with 1.17. APPROVED!!

**Review by qdm12 — APPROVED** [review]

**Review by kishansagathiya — APPROVED** [review]
Change the version here as well before merging

https://github.com/ChainSafe/gossamer/blob/0bc4bf1879f35fbce61d2914dbe0feda21f5651d/docs/docs/getting-started/installation.md#L11

**Pull request merged**